h2gis-utilities/src/test/java/org/h2gis/utilities/JDBCUtilitiesTest.java

```diff
@@ -313,6 +313,119 @@ public void testWrapConnection() {
         assertTrue(JDBCUtilities.wrapConnection(new CustomConnection(connection)) instanceof ConnectionWrapper);
     }
 
+    @Test
+    public void isIndexedTest() throws SQLException {
+        st.execute("DROP TABLE IF EXISTS TEST_INDEX");
+        st.execute("CREATE TABLE TEST_INDEX(no_idx GEOMETRY, idx GEOMETRY, spatial_idx GEOMETRY)");
+        st.execute("CREATE INDEX ON TEST_INDEX(idx)");
+        st.execute("CREATE INDEX ON TEST_INDEX USING RTREE(spatial_idx)");
```

Comment:
It would be better to use different syntax for PostgreSQL and H2 in test cases too here and in other places.

Reply:
Ok, I will keep the `USING RTREE` syntax for the PostgreSQL and use `CREATE SPATIAL INDEX` syntax for H2/H2GIS.

Reply:
Thanks @katzyn for the comments